style.css

```diff
@@ -189,3 +189,92 @@ section .heading-2 {
 #two {
   padding-left: 202px;
 }
+
+.formsection {
+  background-image: url('images/Contact\ me.jpg');
+  width: 300px;
+  margin-top: 300px;
+  background-repeat: no-repeat;
+  height: 779px;
+}
```

Comment:
- Your form is not exactly styled as it is in the design template, the contact form should occupy the entire width of the screen, please fix this 🙏 

Your project:
<img width="207" alt="eeee" src="https://user-images.githubusercontent.com/73607512/200818221-b7504d89-a03c-4c70-b178-9a960382d898.PNG">

Reply:
![image](https://user-images.githubusercontent.com/41912912/200934532-75fbb889-94fb-4d2b-b55b-237ccda261ef.png)

Kindly, this detail is still appearing on your site, please follow the advice of the previous reviewer